allennlp/commands/dry_run.py

```diff
@@ -142,10 +142,4 @@ def dry_run_from_params(params: Params, serialization_dir: str) -> None:
         if any(re.search(regex, name) for regex in no_grad_regexes):
             parameter.requires_grad_(False)
 
-    frozen_parameter_names, tunable_parameter_names = get_frozen_and_tunable_parameter_names(model)
-    logger.info("Following parameters are Frozen  (without gradient):")
-    for name in frozen_parameter_names:
-        logger.info(name)
-    logger.info("Following parameters are Tunable (with gradient):")
-    for name in tunable_parameter_names:
-        logger.info(name)
+    log_and_get_frozen_and_tunable_parameter_names(model)
```

Comment:
Instead of changing this name, I'd add a new function that is something like `log_parameter_tuning`, that calls the existing function.